cli/flox/src/commands/environment.rs

```diff
@@ -1627,17 +1628,24 @@ impl Pull {
                 }
                 .spin()?;
 
-                let complete_message = formatdoc! {"
-                    Pulled {owner}/{name} from {floxhub_host}{suffix}
-
-                    You can activate this environment with 'flox activate'
-                    ",
-                    owner = pointer.owner, name = pointer.name,
-                    floxhub_host = flox.floxhub.base_url(),
-                    suffix = if self.force { " (forced)" } else { "" }
-                };
-
-                message::created(complete_message);
+                match result {
+                    PullResult::Updated => {
+                        message::updated(formatdoc! {"
+                            Pulled {owner}/{name} from {floxhub_host}{suffix}
+
+                            You can activate this environment with 'flox activate'
+                            ",
+                            owner = pointer.owner, name = pointer.name,
+                            floxhub_host = flox.floxhub.base_url(),
+                            suffix = if self.force { " (forced)" } else { "" }
+                        });
+                    },
+                    PullResult::UpToDate => {
+                        message::warning(formatdoc! {"
+                            {owner}/{name} is already up to date.
```

Comment:
nit/question: do you want the trailing newline?

Reply:
It's a bit inconsistent that our `formatdoc!` messages end with a newline and `format!` ones don't, I agree
It's due to how we write them in the source, i.e. `"` on a separate line, so we could change those treewide. but I'd rather do that separately from this PR